sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-student-assessment-status-cols.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE
+  sims.student_assessments
+ADD
+  COLUMN student_assessment_status sims.student_assessment_status NOT NULL DEFAULT 'Created';
+
+COMMENT ON COLUMN sims.student_assessments.student_assessment_status IS 'Student assessment status from its creation till the workflow calculations are finalized or the workflow is cancelled.';
```

Comment:
from it's submission if status is `Submitted` ?

Reply:
I am not sure, the idea was not to target the statuses, for instance, I used `finalized` instead of completed.

Reply:
I am ok.